.github/workflows/create_release.yml

```diff
@@ -8,25 +13,26 @@ on:
 
 jobs:
   release:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-      - name: Check out repository
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Install Dependencies
         run: |
           sudo apt-get update
           sudo apt-get install -y $(find . -name apt-requirements.txt | xargs sed -e "s/#.*//g")
-      - name: Build
+      - name: Create tag
         run: |
-          git tag ${{ inputs.release_tag }}
-          bazel build --stamp :release
```

Comment:
I don't necessarily have a strong opinion but I can see an advantage to using bazel-release in theory: the release process is part of the build and can be done locally easily. Github/Azure flows are always notorioulsy bad for local reproducibility as they get more complicated.

Reply:
That makes sense.

I have this PR for adding bzlmod support to bazel-release: https://github.com/lowRISC/bazel-release/pull/5

If that's merged, we could go back to using it.